dune

```diff
@@ -0,0 +1,5 @@
+(cram
+ (applies_to example)
+ (enabled_if
+  (= true %{env:GITHUB_TESTS=false}))
```

Comment:
I don't understand this: why enable the tests only on the CI?

Reply:
We need a container running with Postgres to execute the PostgreSQL test and an SQLite executable available to run the SQLite test. However, `opam-repo-ci` doesn't support these kinds of tests... This is the only option I've found to ensure we don't break `opam-repo-ci` when releasing new versions.

Reply:
Thanks! I can't come up with a better alternative either, so let's go with this

Reply:
We can correct it later if we find something better :+1: